lighthouse-core/gather/gatherers/js-usage.js

```diff
@@ -16,7 +16,7 @@ class JsUsage extends Gatherer {
    */
   async beforePass(passContext) {
     await passContext.driver.sendCommand('Profiler.enable');
-    await passContext.driver.sendCommand('Profiler.startPreciseCoverage');
+    await passContext.driver.sendCommand('Profiler.startPreciseCoverage', {detailed: false});
```

Comment:
To be certain we're on the same page, is this what was meant by function-level @bmeurer? Or is it only `getBestEffortCoverage` that has the characteristics we're discussing?

Reply:
Yes this gives you function-level coverage. The best effort coverage is not supported since it's unreliable and misleading, and we will likely remove that in the future.